UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/FileShareServiceTests.cs

```diff
@@ -43,7 +43,7 @@ public void Setup()
             this.fakeLogger = A.Fake<ILogger<FileShareService>>();
             this.fakeAuthFssTokenProvider = A.Fake<IAuthFssTokenProvider>();
             this.fakeFileShareConfig = Options.Create(new FileShareServiceConfiguration()
-            { BaseUrl = "http://tempuri.org", CellName = "DE260001", EditionNumber = "1", Limit = 10, Start = 0, ProductCode = "AVCS", ProductLimit = 4, UpdateNumber = "0", UpdateNumberLimit = 10 });
+            { BaseUrl = "http://tempuri.org", PublicBaseUrl = "http://tempuri.org", CellName = "DE260001", EditionNumber = "1", Limit = 10, Start = 0, ProductCode = "AVCS", ProductLimit = 4, UpdateNumber = "0", UpdateNumberLimit = 10 });
```

Comment:
I think it would be preferable if `BaseUrl` and `PublicBaseUrl` were different in at least one test that calls `FileShareService.CreateBatch()`, to verify the URLs are being swapped correctly